### PR TITLE
installation guide for zuri chat desktop

### PR DESCRIPTION
**Pull request by olashina201**

### CODE REVIEW

File: docs/zuri_desktop_installation.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# ZURI CHAT DESKTOP CLIENT INSTALLATION GUIDE
+
+```This guide will walk you through the download and installation process for the Zuri Chat Desktop Client.```
+
+# DOWNLOADING ZURI CHAT DESKTOP CLIENT
+
+- go to the url ``https://zuri.chat/downloads``
```
Comment (Lord-Chris) [inline docs/zuri_desktop_installation.md:7]: Make the link clickable, so it can redirect to the page

File: docs/zuri_desktop_installation.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+- Navigate to the Zuri Chat Desktop
+- Click on any of the download button to download for your operating system.
+  
+  ![picture 7](../images/sc7.png)
```
Comment (Lord-Chris) [inline docs/zuri_desktop_installation.md:11]: The image here doesn't load

File: docs/zuri_desktop_installation.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+# INSTALLATION GUIDE
+
+```Follow the instructions below for successful installation:```
+
```
Comment (Lord-Chris) [inline docs/zuri_desktop_installation.md:16]: Spelling of open here

File: docs/zuri_desktop_installation.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+- Start the installation process by opeing the downloaded file.
+- Specify the path to install the application.
+  
+  ![picture 1](../images/sc1.png)
```
Comment (Lord-Chris) [inline docs/zuri_desktop_installation.md:20]: Image here doesn't open also

**Review by Lord-Chris — CHANGES_REQUESTED** [review]
Great work, but make necessary changes.

Also check all spelling and images

**Comment by olashina201** [comment]
@Lord-Chris all issues resolved

**Review by Lord-Chris — APPROVED** [review]
LGTM

**Pull request merged**